manifests/record/ptr/by_ip.pp

```diff
@@ -102,10 +104,16 @@
     $octet = $class_c_host
   }
 
+  if $zone != undef and $zone != '' {
+    $fqdn = "${host}.${zone}"
+  } else {
+    $fqdn = $host
+  }
+
   dns::record::ptr { "${octet}.${reverse_zone}":
     host => $octet,
     zone => $reverse_zone,
     ttl  => $ttl ,
-    data => "${host}.${zone}"
+    data => $fqdn ,
```

Comment:
Can you remove all these ` ,` (extra spaces before the commas)
I'm not sure there is a specific style guide rule to that, but it just looks kinda different to me